microxrce_agent.cpp

```diff
@@ -158,8 +164,8 @@ int main(int argc, char** argv)
         std::cout << "List of commands:" << std::endl;
         std::cout << "    serial <device_name>" << std::endl;
         std::cout << "    pseudo-serial" << std::endl;
-        std::cout << "    udp <local_port>" << std::endl;
-        std::cout << "    tcp <local_port>" << std::endl;
+        std::cout << "    udp <local_port> [<discovery_port>]" << std::endl;
```

Comment:
According to this help message, discovery port is optional. But in code `atoi(argv[2])` always is done.

Reply:
This is ok, must be argv[2]. But the first `argc == 3` comparation was wrong and is already fixed.